time                  stamp
string[]              tags
float32[]             stamps
kineverse/Operation[] operations
string[]              deleted 